SU-SWS/stanford_person
# D8CORE-1561: Stanford Person

# READY FOR REVIEW

# Summary
- All the stanford_person code

# Needed By (Date)
- May 13th

# Urgency
- Fairly high

# Steps to Test

1. Review code
2. Add this module to a fresh build of the cardinalsites stack on a stanford_profile build
3. Install a fresh stanford_profile install and then enable this module
4. Validate all items show up (views, content types, and default content)
5. Validate that you can create new person nodes and add them to the appropriate places in the view
6. Validate that the views filter correctly and behave as expected
7. PHP tests forthcoming

# Affected Projects or Products
- D8CORE
- SOE & LelandD8

# Associated Issues and/or People
- @rmundstock 
- @jenbreese 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
